diff.go

```diff
@@ -487,7 +487,9 @@ const (
 
 	DiffIgnoreWhitespace       DiffOptionsFlag = C.GIT_DIFF_IGNORE_WHITESPACE
 	DiffIgnoreWhitespaceChange DiffOptionsFlag = C.GIT_DIFF_IGNORE_WHITESPACE_CHANGE
-	DiffIgnoreWitespaceEol     DiffOptionsFlag = C.GIT_DIFF_IGNORE_WHITESPACE_EOL
+	DiffIgnoreWhitespaceEOL    DiffOptionsFlag = C.GIT_DIFF_IGNORE_WHITESPACE_EOL
+	// Deprecated: use DiffIgnoreWhitespaceEOL instead
+	DiffIgnoreWitespaceEol = DiffIgnoreWhitespaceEOL
```

Comment:
Can this be moved to `deprecated.go`? that way everything that's deprecated is in the same file.